hack/machineset.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+
+# machineset.sh - apply/delete the Windows MachineSet for Windows Machine Config Operator
+#
+# USAGE
+#    machineset.sh apply/delete
+# OPTIONS
+#    $1      Action       apply/delete the MachineSet
+# PREREQUISITES
+#    oc                   to apply MachineSets on the cluster(cluster should be logged in)
+#    aws                  to fetch Windows AMI id for AWS platform (only required for clusters running on AWS)
+
+ACTION=$1
```

Comment:
I suggest the script should just generate the yaml. Applying / deleting should be left to the user through oc. This will also allow us to make changes to it before applying.

Reply:
Sorry, forgot to update. 
I've made the change which creates a YAML file if no action is provided and will create a machineset directly if an action is provided. 
This due to the fact that, we wont be changing the machineset always and will not result in an additional file